music/src/ddsp/spice.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ *
+ * @license
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as tf from '@tensorflow/tfjs';
+import { SPICE_MODEL_URL } from './constants';
+
+async function startSpice() {
```

Comment:
Yah, if you follow the same general model class API that the rest of magenta.js uses for the spice model as well, I think it would be clearer how to use it and more modular.